src/Utilities.cpp

```diff
@@ -283,125 +283,125 @@ double brentRoot(
   double lowerBound,
   double upperBound,
   std::function<double(double)> func,
-  double epsilon) {
-    double counterPoint = lowerBound;
-    double currentPoint = upperBound;
-    double counterFunc = func(counterPoint);
-    double currentFunc = func(currentPoint);
-    if (counterFunc * currentFunc > 0.0) {
-      throw std::invalid_argument("Function values at the boundaries have the same sign [brentRoot].");
-    }
-    if (fabs(counterFunc) < fabs(currentFunc)) {
-      std::swap(counterPoint, currentPoint);
-      std::swap(counterFunc, currentFunc);
-    }
-
-    double previousPoint = counterPoint;
-    double previousFunc = counterFunc;
-    double evenOlderPoint = previousPoint;
-    double nextPoint;
-    double nextFunc;
-    int iteration = 0;
-    bool bisected = true;
-
-    do {
-      // Inverse quadratic interpolation
-      if (counterFunc != previousFunc && counterFunc != currentFunc && currentFunc != previousFunc) {
-        nextPoint = (counterPoint * currentFunc * previousFunc) / ((counterFunc - currentFunc) * (counterFunc - previousFunc));
-        nextPoint += (currentPoint * counterFunc * previousFunc) / ((currentFunc - counterFunc) * (currentFunc - previousFunc));
-        nextPoint += (previousPoint * currentFunc * counterFunc) / ((previousFunc - counterFunc) * (previousFunc - currentFunc));
-      }
-      // Secant method
-      else {
-        nextPoint = currentPoint - currentFunc * (currentPoint - counterPoint) / (currentFunc - counterFunc);
-      }
-
-      // Bisection method
-      if (((currentPoint - nextPoint) * (nextPoint - (3 * counterPoint + currentPoint) / 4) < 0) ||
-          (bisected && fabs(nextPoint - currentPoint) >= fabs(currentPoint - previousPoint) / 2) ||
-          (!bisected && fabs(nextPoint - currentPoint) >= fabs(previousPoint - evenOlderPoint) / 2) ||
-          (bisected && fabs(currentPoint - previousPoint) < epsilon) ||
-          (!bisected && fabs(previousPoint - evenOlderPoint) < epsilon)) {
-        nextPoint = (currentPoint + counterPoint) / 2;
-        bisected = true;
-      }
-      else {
-        bisected = false;
-      }
-
-      // Setup for next iteration
-      evenOlderPoint = previousPoint;
-      previousPoint = currentPoint;
-      previousFunc = currentFunc;
-      nextFunc = func(nextPoint);
-      if (counterFunc * nextFunc < 0) {
-        currentPoint = nextPoint;
-        currentFunc = nextFunc;
-      }
-      else {
-        counterPoint = nextPoint;
-        counterFunc = nextFunc;
-      }
-    } while (++iteration < 30 && fabs(counterPoint - currentPoint) > epsilon);
+  double epsilon)
+{
```

Comment:
Do we want this curly-brace placement to be standard for usgscsm, and if so, how do you feel about me adding a ticket to update the framer code to use this standard?

Reply:
We don't really have a standard right now. I had run a linter over the code once, but never committed it. I'm not sure if I like this or now. Maybe I should just change these?

Reply:
I don't know either. I was honestly just thinking that if we have hours left and get all the actual SAR work done, it might be a nice bonus to decide on one way or the other and then standardize the code. (If I have time, I volunteer!)

Reply:
As far as I can tell, Ls and Sar use:
```
double func(int x)
{
}
```

and framer uses: 

```
double func(int x){
}
```
I'm happy to call this "beyond the scope of this PR" and talk about this later (if we do end up having time.)